tests/unit/reaction/test_combinatorics.py

```diff
@@ -101,14 +99,13 @@ def test_constructor():
         assert representation.final_state == [["gamma", "pi0"]]
 
     @staticmethod
-    def test_from_graph(three_body_decay, particle_database):
-        graph1 = StateTransitionGraph.from_topology(three_body_decay)
-        pion = particle_database["pi0"]
-        graph1.edge_props[0] = ("J/psi", [-1, +1])
-        graph1.edge_props[2] = pion
-        graph1.edge_props[3] = pion
-        graph1.edge_props[4] = "gamma"
-        kinematic_representation1 = _get_kinematic_representation(graph1)
+    def test_from_topology(three_body_decay):
+        pi0 = ("pi0", [0])
```

Comment:
Is it required to specify the spin projections now?

Reply:
was that different before?

Reply:
I would have expected `"pi0"` would suffice. But actually the test has become a bit different: it now uses this tuple in the kinematic representation, previously it was an instance of `Particle`. But here, both are fine, they are just used to compare the permutations.